test/pages/authentication_page.rb

```diff
@@ -1,15 +1,34 @@
 module AuthenticationPage
   def sign_in(user)
-    if page.driver.respond_to?(:basic_auth)
-      page.driver.basic_auth(user.email, user.password)
-    elsif page.driver.respond_to?(:basic_authorize)
-      page.driver.basic_authorize(user.email, user.password)
-    elsif page.driver.respond_to?(:browser) && page.driver.browser.respond_to?(:basic_authorize)
-      page.driver.browser.basic_authorize(user.email, user.password)
-    else
-      raise "I don't know how to sign in!"
-    end
+    OmniAuth.config.add_mock(:github, info: {
+      nickname: user.nickname, image: user.avatar_url, email: user.email
+    }) unless @will_forget_credentials
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.